src/Interface/Lists/Radarcord.ts

```diff
@@ -66,11 +66,70 @@ export default class RadarBotDirectory extends Service {
   getBotVotes(botID: IDResolvable, userID: IDResolvable) {
     return this._request(
       {
-        url: `/lastvoted/${Util.resolveID(userID)}/${Util.resolveID(botID)}`,
-        headers: { Authorization: this.token }
+        url: `/lastvoted/${Util.resolveID(userID)}/${Util.resolveID(botID)}`
```

Comment:
Is this endpoint still in the API? Where is it documented in the docs?
Also, it's using the same name as below

Reply:
It's an API endpoint, just reminded me to add it to the documentation.
Should be called lastVoted() or something similar

Reply:
Yup, `lastVoted()` will be fine 👍🏻